src/database.js

```diff
@@ -40,16 +40,15 @@ async function dropColumnsIfExist (knex, tableName, columns) {
   })
 }
 
-async function createSchema ({ connection, schema, destroyKnex = true }) {
-  const params = {
-    client: 'pg',
-    useNullAsDefault: true,
-    connection,
-    searchPath: [schema]
-  }
-
-  // get a new connection because we have to set schema if needed
-  const knex = Knex(params)
+/**
+ * @param {Object}  connection - provide `connection` or `knex`
+ * @param {Object}  knex - Knex.js query builder
+ * @param {String}  schema
+ * @param {String}  [destroyKnex = true] - if false, knex isn't destroyed so it can be reused
+ * @return {Object|Undefined} returns knex if `destroyKnex` is false
+ */
+async function createSchema ({ connection, schema, knex, destroyKnex = true }) {
+  if (!knex) knex = getKnex({ connection, schema })
```

Comment:
Why destroy by default if the function is able to create its own short-lived knex instance?

Reply:
Right, I'll change that.